src/uint256.cpp

```diff
@@ -33,7 +33,7 @@ void base_blob<BITS>::SetHex(const char* psz)
         psz++;
 
     // skip 0x
-    if (psz[0] == '0' && tolower(psz[1]) == 'x')
+    if (psz[0] == '0' && ToLower((unsigned char)psz[1]) == 'x')
```

Comment:
It's kind of strange for a `ToLower` function to take an unsigned char (because `std::string`s index as `char`), but that's not introduced in this PR.